app/src/androidTest/java/com/github/sdp/ratemyepfl/activity/SplashScreenTest.kt

```diff
@@ -28,8 +31,9 @@ class SplashScreenTest {
         init()
         FakeConnectedUser.loggedIn = true
         val intent = Intent(ApplicationProvider.getApplicationContext(), SplashScreen::class.java)
+        //intent.putExtra("source", "test")
         val scenario: ActivityScenario<SplashScreen> = ActivityScenario.launch(intent)
-        intended(toPackage("com.github.sdp.ratemyepfl"))
+        intended(toPackage("com.github.sdp.ratemyepfl"), times(2))
```

Comment:
To avoid using times(2) you can maybe try to make the package more specific (maybe "com.github.sdp.ratemyepfl.activity" or use toComponent.

Reply:
I did, and sadly it does not work. The intent has com._.ratemyepfl/.activity, and it only matches with "com.github.sdp.ratemyepfl". I don't know why, but the intent has component cmp=com.github.sdp.ratemyepfl/.activity.MainActivity instead of the class name, so I can't apply it either.